jpegenc/models/rgb2ycbcr/test_rgb2ycbcr.py

```diff
@@ -0,0 +1,148 @@
+#!/bin/python
+from myhdl import StopSimulation, block, Signal, ResetSignal, intbv, delay, instance, always_comb, always_seq
+from random import randrange
+from rgb2ycbcr import *
+from myhdl.conversion import verify
+
+def rgb_to_ycbcr(r, g, b):
+
+    ycbcr = [None for _ in range(3)]
+    ycbcr[0] = int(
+        round(Y_COEFF[0] *
```

Comment:
@cfelton definetely the code needs PEP8 refactoring. I will do it in the following commits.

Although that my starting code was based on the older version of this file https://github.com/mkatsimpris/test_jpeg/blob/master/hdl/jpegenc_v2/myhdl/rgb2ycbcr.py the final version is completely different. More modular and correct I think. 

I can start from scratch if you think so.